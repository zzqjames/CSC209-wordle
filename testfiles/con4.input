y 1 -yy-- -g-g- ditty
y 2 -yy-- -g-g- ditty