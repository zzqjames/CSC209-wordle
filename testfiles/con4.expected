cannot_be: 
must_be
[0] 
[1] t 
[2] i t 
[3] 
[4] 
